ui/pages/settings/networks-tab/custom-content-search/custom-content-search.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import TextField from '../../../../components/ui/text-field';
+import CustomContentSearch from './custom-content-search';
+
+describe('CustomContentSearch', () => {
+  let wrapper;
+
+  it('should render custom content search correctly', () => {
+    wrapper = shallow(
+      <CustomContentSearch onSearch={() => undefined} networksList={[]} />,
+      {
+        context: {
+          t: (s) => `${s}`,
+        },
+      },
+    );
+
+    expect(wrapper.find(TextField).props().id).toStrictEqual('search-networks');
+    expect(wrapper.find(TextField).props().value).toBeUndefined();
+  });
+});
```

Comment:
can we add tests that the onSearch function is called when TextField value changed.

Reply:
@segun Done [here](https://github.com/MetaMask/metamask-extension/pull/13560/commits/98e09b9d4634bac67e9d41910a224f8b3157aa19)